core/src/actions/oAuth.ts

```diff
@@ -47,6 +56,8 @@ export class OAuthClientStart extends CLSAction {
       id: params.id,
       type: params.type,
       provider: params.provider,
+      appId: params.appId,
+      appOption: params.appOption,
```

Comment:
These maybe aren't in use?

Reply:
`appOption` is being used, but `appId` isn't really. Was going to remove it but thought it couldn't hurt to have just in case, to know which app that `appOption` is referring to